src/parser/DASHTree.cpp

```diff
@@ -1775,19 +1775,54 @@ void DASHTree::RefreshLiveSegments()
 
       for (size_t index{0}; index < updateTree->periods_.size(); index++)
       {
-        if (index == periods_.size()) // Exceeded periods
-          break;
         auto updPeriod = updateTree->periods_[index];
         if (!updPeriod)
           continue;
 
+        Period *period{nullptr};
```

Comment:
every time you write new code always apply Clang format

on this project we dont have Jenkins integrations
so you have to apply clang format manually so by using your editor shourcuts,
for example on visual studio CTRL+K+F

Reply:
:+1: thanks